### ISSUE TITLE
Research Facility Locator Search for "Find a Rep" Integration

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#65578]
### Issue Description
The aim is to understand the workings of the Facility Locator search functionality, especially when it comes to utilizing partial address information like just the zip-code. This is essential as we attempt to implement a location-based search for representatives using only the zip-code data available for each representative. Additionally, we need to determine how to correlate non-zip code searches, such as by city or state, to available zip codes.

---

### Tasks
- [x] Investigate and document how the Facility Locator search endpoint operates on VA.gov.
- [ ] ~~Determine and document the methods to correlate non-zip code search entries (e.g., ‘Williston, VT’, ‘VT’, ‘Vermont’) to actual zip-codes.~~
- [ ] ~~Assess what components of the Facility Locator code can be reused. Generate a list of files that can be copied and modified for the "Find a Rep" functionality.~~
- [ ] ~~Identify search parameters from the Facility Locator code that will not be necessary for "Find a Rep" and therefore, can be removed.~~
- [ ] ~~Determine the additional search parameters that will need to be added to repurpose the Facility Locator code for "Find a Rep."~~
- [ ] ~~Collaborate with the frontend, referencing the related ticket ([#66041](https://app.zenhub.com/workspaces/accredited-representation-management-team-64d0dc51d3e8f4788ac6ef96/issues/gh/department-of-veterans-affairs/va.gov-team/66041)), to ensure alignment regarding the required URL query parameters.~~

### Acceptance Criteria
- [x] A comprehensive technical document is available that provides insights into the integration of Facility Locator search with the representative search. This document should address the challenges of using only zip-code data and also cover the potential of correlating city or state searches to zip codes.

**Comment by jvcAdHoc** [comment]
[Initial research](https://docs.google.com/document/d/1cj2HlstFu1J6ZlBNssr9FNx8qi7C5xEKZg47mf-Ea-c/edit)

**Comment by jvcAdHoc** [comment]
Initial research found that the Find a VA Location feature can hit two different API endpoints in vets-api depending on the type of facility being search for: [va_controller](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/facilities_api/app/controllers/facilities_api/v1/va_controller.rb) or [ccp_controller](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/facilities_api/app/controllers/facilities_api/v1/ccp_controller.rb). The `va_controller` acts almost like a straight pass through to [lighthouse](https://developer.va.gov/explore/api/va-facilities) and `ccp_controller` makes requests to `dws`.

Vets-api doesn't handle the geospatial search functionality. That responsibility lives within the APIs vets-api reaches out to. That information coupled with us not building rep search functionality under the lighthouse umbrella, we are going to implement this using postgis within vets-api. The pattern exists for [vha-facilities](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/facilities/vha_facility.rb).

See #66522

tldr: A "Facilities Locator Search" integration for Find a Rep does not make sense and we are pivoting to a postgis solution on the backend
